src/lib/org_utils.js

```diff
@@ -657,3 +657,6 @@ export const getTodoKeywordSetsAsFlattenedArray = (state) => {
     .toSet()
     .toJS();
 };
+
+/** Regular expression of file extensions to validate a filename. */
+export const orgFileExtensions = /org$|org_archive$/;
```

Comment:
I'm not fully convinced of enforcing a filename extension for org files. But anyway, if we do, it should be an extension starting with `.`.

Reply:
For example, my emacs or whatever often creates files like `#my.org#` or `my.org~`. Or maybe a beginner saves a file as `.txt`. Is it a problem to just show the "cannot parse" error message for non-org files?

Reply:
Good point; fixed in the latest commit.

Reply:
> Why should we prohibit to open these files?

Your examples sound a lot like tmp files for unsaved buffers. I’d say it’s prudent not to open these, because any changes to it are potentially just going to the GC^^

My initial reason to limit for those two extensions was not links, though. I’m filtering all files in the file explorer to be of that type or of a folder. The rationale here is that I personally have about two zillion of files - and when using organice, I don’t care for any of them except for the ones that I can actually work on. 

There’s certainly other options for that:

- more powerful file explorer (search/sort/filter)
- make the file extensions configurable

Reply:
OK, makes sense.